config/software/pyopenssl.rb

```diff
@@ -1,18 +1,32 @@
 name "pyopenssl"
+# If you upgrade pyopenssl, you'll probably have to upgrade `cryptography` as well
 default_version "0.14"
 
-dependency "openssl"
-dependency "python"
-dependency "pip"
-dependency "libffi"
+if ohai['platform'] == 'windows'
+  dependency "openssl-windows"
+  dependency "python"
+  dependency "pip"
+  dependency "cryptography"
 
-build do
-  ship_license "https://raw.githubusercontent.com/pyca/pyopenssl/master/LICENSE"
-  build_env = {
-    "PATH" => "/#{install_dir}/embedded/bin:#{ENV['PATH']}",
-    "LDFLAGS" => "-L/#{install_dir}/embedded/lib -I#{install_dir}/embedded/include",
-    "LD_RUN_PATH" => "#{install_dir}/embedded/lib",
-    "CFLAGS" => "-L#{install_dir}/embedded/lib -I#{install_dir}/embedded/include/"
-  }
-  command "#{install_dir}/embedded/bin/pip install pyOpenSSL==#{version}", :env => build_env
+  build do
+    ship_license "https://raw.githubusercontent.com/pyca/pyopenssl/master/LICENSE"
+    # No -I because we want to use the already installed version of `cryptography`
+    pip "install pyopenssl==#{version}"
+  end
+else
+  dependency "openssl"
+  dependency "libffi"
+  dependency "python"
+  dependency "pip"
+
```

Comment:
we probably want to add `cryptography` as a dependency for unix too (looks like it's a dep on all platforms, see https://github.com/pyca/pyopenssl/blob/0.14/setup.py#L37)